CHANGELOG.md

```diff
@@ -43,7 +43,7 @@ All notable changes to `twill` will be documented in this file.
 - Rename default migrations to prevent conflicts in existing codebases [`74173acd`](https://github.com/area17/twill/commit/74173acd0b750243c2d3bab382ed842f36efdce9)
 
 #### Frontend build
-- *@antonin and @ifox*
```

Comment:
Let's remove this commit `c35a20346e240415d9589fceb14fa7df9165d6c6` from the pull request.